File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/NetworkConfig.java
package com.syahputrareno975.cardbattlemodule.model;
import java.io.Serializable;

public class NetworkConfig implements Serializable {
  public String Url = "";
  public int Port = 0;

  public NetworkConfig(String url, int port) {
    Url = url;
    Port = port;
  }

  public String toHttpUrl() {
    return "http://" + this.Url + ":" + this.Port;
  }

}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/NetworkConfig.kt
package com.syahputrareno975.simplecardbattle.model

import java.io.Serializable

class NetworkConfig : Serializable {
  var Url = ""
  var Port = 0

  constructor(Url: String, Port: Int) {
    this.Url = Url
    this.Port = Port
  }
}